src/switchroot/ostree-relabel-etc.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2011 Colin Walters <walters@verbum.org>
```

Comment:
Tangentially related, I think you can update this to be you and or/alex, or just say (C) Red Hat, Inc.